include/dxc/DXIL/DxilConstants.h

```diff
@@ -983,9 +1028,9 @@ enum class OpCode : unsigned {
   NumOpCodes_Dxil_1_5 = 216,
   NumOpCodes_Dxil_1_6 = 222,
   NumOpCodes_Dxil_1_7 = 226,
-  NumOpCodes_Dxil_1_8 = 258,
+  NumOpCodes_Dxil_1_8 = 303,
```

Comment:
These are DXIL 1.9 opcodes not 1.8.

Reply:
Fixed in https://github.com/microsoft/DirectXShaderCompiler/pull/7125/commits/8d93c0619a739b44120104c50d9bf964adc8ff93